tests/cli/RetryCest.php

```diff
@@ -31,8 +31,8 @@ public function failForTime(CliGuy $I)
     public function failForInterval(CliGuy $I)
     {
         $I->executeFailCommand('run --debug -g fail2');
-        $I->seeInShellOutput('Retrying #1 in 100ms');
         $I->seeInShellOutput('Retrying #2 in 200ms');
+        $I->seeInShellOutput('Retrying #1 in 100ms');
         $I->seeInShellOutput('Retrying #3 in 400ms');
```

Comment:
Have you changed the order of these assertions by accident?
Please change it back.

Reply:
Ouch, I thought I didn't...